scripts/funcotator/data_sources/gnomAD/createGnomadAlleleFreqTsv.wdl

```diff
@@ -0,0 +1,114 @@
+#     Int  mem                       -  Amount of memory to give to the machine running each task in this workflow.
+#     Int  preemptible_attempts      -  Number of times to allow each task in this workflow to be preempted.
+#     Int  disk_space_gb             -  Amount of storage disk space (in Gb) to give to each machine running each task in this workflow.
+#     Int  cpu                       -  Number of CPU cores to give to each machine running each task in this workflow.
+#     Int  boot_disk_size_gb         -  Amount of boot disk space (in Gb) to give to each machine running each task in this workflow.
+#
+# This WDL needs to decide whether to use the ``gatk_jar`` or ``gatk_jar_override`` for the jar location.  As of cromwell-0.24,
+# this logic *must* go into each task.  Therefore, there is a lot of duplicated code.  This allows users to specify a jar file
+# independent of what is in the docker file.  See the README.md for more info.
+#
+workflow CreateGnomadAlleleFreqTsv {
+
+    File gnomAD_file = "gs://gnomad-public/release/2.1/vcf/genomes/gnomad.genomes.r2.1.sites.vcf.bgz"
+    String out_file_name
+
+    String gatk_docker
+
+    Int?  mem_gb
+    Int? preemptible_attempts
+    Int? disk_space_gb
+    Int? cpu
+    Int? boot_disk_size_gb
+
+    call CreateGnomadAlleleFreqTsvTask {
+        input:
+            gnomAD_file               = gnomAD_file,
+            out_file_name             = out_file_name,
+            gatk_docker               = gatk_docker,
+            mem_gb                    = mem_gb,
+            preemptible_attempts      = preemptible_attempts,
+            disk_space_gb             = disk_space_gb,
+            cpu                       = cpu,
+            boot_disk_size_gb         = boot_disk_size_gb
+    }
+
+    output {
+        File vcf_file = CreateGnomadAlleleFreqTsvTask.tsvOut
+    }
+}
+
+
+task CreateGnomadAlleleFreqTsvTask {
+
+    File gnomAD_file
+    String out_file_name
+
+     # ------------------------------------------------
+     # runtime
+     String gatk_docker
+     Int? mem_gb
+     Int? preemptible_attempts
+     Int? disk_space_gb
+     Int? cpu
+     Int? boot_disk_size_gb
+
+     # ------------------------------------------------
+     # Get machine settings:
+     Boolean use_ssd = false
+
+     # You may have to change the following two parameter values depending on the task requirements
+     Int default_ram_mb = 1024 * 3
+     # WARNING: In the workflow, you should calculate the disk space as an input to this task (disk_space_gb).  Please see [TODO: Link from Jose] for examples.
+     Int default_disk_space_gb = 100
+
+     Int default_boot_disk_size_gb = 15
+
+     # Mem is in units of GB but our command and memory runtime values are in MB
+     Int machine_mem = if defined(mem_gb) then mem_gb * 1024 else default_ram_mb
+     Int command_mem = machine_mem - 1024
```

Comment:
Typically, command_mem is only needed if you are running a java command (or similar) as opposed to machine_mem which is the RAM on the machine.

Reply:
Yeah - I have a template I use that has all these values in it.  

I'll remove it.